executable/FnmApp.re

```diff
@@ -74,6 +75,23 @@ let install = {
   );
 };
 
+let uninstall = {
+  let doc = "Uninstall a node version";
+  let man = [];
+
+  let selectedVersion = {
+    let doc = "Uninstall the node version specified in $(docv).";
+    Arg.(
+      value & pos(0, some(string), None) & info([], ~docv="VERSION", ~doc)
```

Comment:
by changing `value` to `required`, you get type checking for free:

```reason
      required & pos(0, some(string), None) & info([], ~docv="VERSION", ~doc)
```

then you can have `Uninstall.re` to expose a `(~version: string) => Lwt.t` function instead of `(~version: option(string)) => Lwt.t` function 😄

Reply:
Thx it's indeed more clear like that